app/assets/javascripts/discourse/tests/acceptance/route-control-test.js

```diff
@@ -0,0 +1,117 @@
+import {
+  acceptance,
+  exists,
+  updateCurrentUser,
+} from "discourse/tests/helpers/qunit-helpers";
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import { extraConnectorClass } from "discourse/lib/plugin-connectors";
+import { hbs } from "ember-cli-htmlbars";
+
+const PREFIX = "javascripts/single-test/connectors";
+
+acceptance("Route Control (as visitor)", function (needs) {
+  needs.settings({
+    top_menu: "latest|categories|top|bookmarks",
+  });
+  needs.hooks.beforeEach(() => {
+    extraConnectorClass("below-site-header/hello");
+
+    // eslint-disable-next-line no-undef
+    Ember.TEMPLATES[`${PREFIX}/below-site-header/hello`] = hbs`
```

Comment:
Since this PR was created we've removed the use of `Ember.TEMPLATES`. Once rebased on `main`, you'll need to switch to use the `registerTemporaryModule` helper. e.g.

https://github.com/discourse/discourse/blob/b96869d5fb7702f74b8191d3fb0108d68ff0df51/app/assets/javascripts/discourse/tests/integration/template-override-test.js#L36-L39